hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/TestHoodieIndex.java

```diff
@@ -77,7 +79,14 @@
 
   private void setUp(IndexType indexType) throws Exception {
     this.indexType = indexType;
-    initResources();
+    initPath();
```

Comment:
is it possible to change `initResources()` to take a properties file?

Reply:
Love to not expand these setup blocks again. our dear Raymond has spent a lot of time cleaning them up :)